src/Stratis.Bitcoin.Features.Miner/MiningFeature.cs

```diff
@@ -162,6 +162,9 @@ public static IFullNodeBuilder AddMining(this IFullNodeBuilder fullNodeBuilder,
             {
                 features
                     .AddFeature<MiningFeature>()
+                    .DependOn<MempoolFeature>()
```

Comment:
Mining I would say probably also depends on Consensus and Store

Reply:
I didn't add them because Stratis.Bitcoin.Features.Miner project don't have references to them.
Would you like me to reference them if order to make change you've mentioned?

Reply:
Mempool has a reference right?
I think we should one will never mine without running a fullnode.

Reply:
>Mempool has a reference right?
Yes. 
Miner references Stratis.Bitcoin, MemoryPool, RPC and Wallet.